sambacc/inotify_waiter.py

```diff
@@ -85,7 +85,7 @@ def _get_events(self):
             and ((event.mask & _inotify.flags.CLOSE_WRITE) != 0)
         ]
 
-    def _wait(self):
+    def _wait(self) -> typing.Iterator[None]:
```

Comment:
Why no just have a single `wait` function? What is the advantage of using `next(typing.Iterator[None])` ?

Reply:
At the time I must have felt implementing the function in terms of an iterator was easier or clearer. This PR only adds the proper type annotation to the function, it doesn't change the runtime behavior. Let's leave that for another time.